frontend/src/Store/Actions/releaseActions.js

```diff
@@ -276,8 +242,7 @@ export const reducers = createHandleActions({
     return newState;
   },
 
-  [SET_RELEASES_SORT]: createSetClientSideCollectionSortReducer(section),
-  [SET_EPISODE_RELEASES_FILTER]: createSetClientSideCollectionFilterReducer(episodeSection),
-  [SET_SEASON_RELEASES_FILTER]: createSetClientSideCollectionFilterReducer(seasonSection)
+  [SET_MOVIE_RELEASES_FILTER]: createSetClientSideCollectionFilterReducer(section),
```

Comment:
Can just be SET_RELEASES_FILTER, we don’t have multiple entities, so don’t have to differentiate.